.github/workflows/ci.yml

```diff
@@ -27,9 +27,10 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -r ${{ matrix.path }}/requirements.txt --no-cache-dir
           pip uninstall -y jina
           pip install jina[cicd,test,http,hub] requests --no-cache-dir
+          pip install -r ${{ matrix.path }}/requirements.txt --no-cache-dir
```

Comment:
I think the point is to make sure `jina` is installed from latest version to keep track if examples work with new versions of jina

Reply:
How you think would be the good way to go around this then? Because the requirements are being overwritten which causes the conflicts

Reply:
maybe a clear exception for pokedex with installing tf 2.1.0

Reply:
Adding something like this at the end of the other installs?
`pip install -r pokedex-with-bit/requirements.txt --no-cache-dir`